CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,6 @@
+# Code of Conduct
+
+We follow the [CNCF Code of Conduct](https://github.com/cncf/foundation/blob/main/code-of-conduct.md).
+
+Please the [CNCF Code of Conduct Committee](mailto:conduct@cncf.io)
```

Comment:
Need to rephrase this sentence... 
Something like: Instances of abusive, harassing, or otherwise unacceptable behavior may be reported by contacting to.....

Reply:
It's missing a word. Should be `Please *contact* the ...`

Reply:
I copied it over from their sample project repo files. Until we decide on our own reporting email.